devutils/src/devutils/invoke_utils.py

```diff
@@ -217,21 +251,22 @@ def devinstall(ctx):
             for pyproject in all_pyprojects:
                 roundtrip_py_project = RoundtripPyProject(pyproject)
                 roundtrips.append(roundtrip_py_project)
+
                 with roundtrip_py_project.update() as contents:
                     dependencies = contents["tool"]["poetry"]["dependencies"]
 
                     for key, value in tuple(dependencies.items()):
+                        if not key.startswith("robocorp-"):
+                            continue
+
                         # Changes something as:
                         # robocorp-log = "0.1.0"
                         # to:
-                        # robocorp-log = {path = "../log/", develop = true}
-                        if key.startswith("robocorp-"):
-                            name = key[len("robocorp-") :]
-                            value = dict(path=f"../{name}/", develop=True)
-                            dependencies[key] = value
-
-            poetry(ctx, "lock --no-update")
-            poetry(ctx, "install")
+                        # robocorp-log = {path = "../log/", develop = true
+                        name = key[len("robocorp-") :]
+                        if all or name in projects:
+                            dependencies[key] = dict(path=f"../{name}/", develop=True)
```

Comment:
🐛 Bug spotted:
```suggestion
                        if all or key in projects:
                            dependencies[key] = dict(path=f"../{name}/", develop=True)
```

- `name`: `workitems`
- `key`: `robocorp-workitems`
- `projects`: [`robocorp-workitems`]

So you should check the `key` belonging to the list, not just the `name`, which always fails to be found.

Reply:
projects is actually the list we pass on which is `workitems, tasks` and so on. I think it would be annoying to write the `robocorp-` suffix for all of our packages we want to install in devmode.

Reply:
Yes it is, for some reason, during testing, I thought it doesn't do the job if you pass `-l workitems` for e.g., so I fooled myself into thinking that's a bug when I tried with `-l robocorp-workitems`.

To be on the safe side, you can apply a `projects = [package.replace("robocorp-", "").strip() for package in (local or "").split(",")]` so no matter if somebody passes it with `robocorp-` prefix or not, it would still work.